js-datatypes/answer.js

```diff
@@ -29,8 +29,17 @@ function submission() {
     // put your answers here in strings, separated by comma
     // the first answer is provided as an example
       "number",
-      
-    // YOUR CODE HERE
+      "string",
+      "object",
+      "string",
+      "string",
+      "number",   //but I think it may be not "number" because typeof(+"run")  also equal to "number"
+      //I google and think so:
+      //I have find "number" include NaN(stand for"not a number"),so if the values in the form of expression, it is treated as a number, but it cannot actually perform any arithmetic operations
+      "boolean",
+      "boolean",
+      "number",
+      "string",
   ];
 
   return answers;
```

Comment:
You might be logically right, dear. Unfortunately, the computer does not agree with us - attached are the tests for that one. 

![Screen Shot 2021-03-27 at 10 50 20](https://user-images.githubusercontent.com/48189833/112708021-64d36880-8eea-11eb-8bc3-09580a8957e9.png)
<img width="368" alt="Screen Shot 2021-03-27 at 10 49 51" src="https://user-images.githubusercontent.com/48189833/112708024-66049580-8eea-11eb-9184-8d06528a5abb.png">